op-challenger/cmd/main_test.go

```diff
@@ -59,15 +61,14 @@ func TestL1ETHRPCAddress(t *testing.T) {
 	})
 }
 
-func TestAlphabetTrace(t *testing.T) {
+func TestTraceType(t *testing.T) {
```

Comment:
We should have a test for invalid trace types too, and check that all valid options are accepted.